Sources/NIOSSL/SwiftCrypto/NIOSSLSecureBytes.swift

```diff
@@ -262,7 +262,7 @@ extension NIOSSLSecureBytes.Backing {
 
     /// Appends the bytes of a slice of another backing buffer to this storage, crashing if there
     /// is not enough room.
-    /* private but inlinable */ func _appendBytes(_ backing: NIOSSLSecureBytes.Backing, inRange range: Range<Int>) {
+    /* private but inlinable */ public func _appendBytes(_ backing: NIOSSLSecureBytes.Backing, inRange range: Range<Int>) {
```

Comment:
This should be `@inlinable`, not public,

Reply:
resolved in [7742aa0](https://github.com/apple/swift-nio-ssl/pull/369/commits/7742aa0cc15f7b191af71578fd2227c7041a7cba)